### ISSUE TITLE
Error while generating Word Wise

### ISSUE THREAD

**Issue by datthinh1801** [xxyzz/WordDumb#3]
Hi, it's me again 😂
Recently, I've installed WordDumb on my younger sister's Windows 10 computer and encountered this error while performing WordDumb on a book.  

```
calibre, version 5.18.0 (win32, embedded-python: True)
Dumb error: Failed: Generating Word Wise for Atomic Habits

Starting job: Generating Word Wise for Atomic Habits 
Job: "Generating Word Wise for Atomic Habits" failed with error: 
Traceback (most recent call last):
  File "calibre\gui2\threaded_jobs.py", line 83, in start_work
  File "calibre_plugins.worddumb.parse_job", line 17, in do_job
  File "calibre_plugins.worddumb.unzip", line 26, in install_libs
  File "calibre_plugins.worddumb.unzip", line 88, in pip_install
  File "subprocess.py", line 364, in check_call
subprocess.CalledProcessError: Command '['python3', '-m', 'pip', 'install', '-t', WindowsPath('C:/Users/user/AppData/Roaming/calibre/plugins/worddumb-libs/nltk3.6.2'), 'nltk==3.6.2']' returned non-zero exit status 9009.
 
Called with args: ((2, 'MOBI', 'BBOOWNQWA7', 'C:\\Users\\user\\Calibre Library\\James Clear\\Atomic Habits (2)\\Atomic Habits - James Clear.mobi', <calibre.ebooks.metadata.book.base.Metadata object at 0x000001A8248C59A0>), True) {'notifications': <queue.Queue object at 0x000001A8248C5400>, 'abort': <threading.Event object at 0x000001A8248C12E0>, 'log': <calibre.utils.logging.GUILog object at 0x000001A8248C1670>}
```  

After running WordDumb and facing this error, I check the `C:/Users/user/AppData/Roaming/calibre/plugins/` directory but don't see `worddumb-libs` directory there.  

FYI: I've tried installing `nltk 3.6.2` manually but it didn't help, the same error still occurs.

**Comment by xxyzz** [comment]
Can you post the output of these commands?

```
$ python3 -m pip --version
$ python3 -m pip install -t C:/Users/user/AppData/Roaming/calibre/plugins/worddumb-libs/nltk3.6.2 nltk==3.6.2
```

**Comment by datthinh1801** [comment]
It works properly new. Thank you so much!

**Comment by xxyzz** [comment]
How did you solve the error? The `pip install` didn't throw an error when you manually run it?

**Comment by datthinh1801** [comment]
Initially, I installed `nltk` manually from pypi but WordDumb still threw the same exception. Then I ran your command `python3 -m pip install -t C:/Users/user/AppData/Roaming/calibre/plugins/worddumb-libs/nltk3.6.2 nltk==3.6.2` explicitly from command prompt and it worked.  
> The actual command that I ran was `python -m pip install -t C:/Users/user/AppData/Roaming/calibre/plugins/worddumb-libs/nltk3.6.2 nltk==3.6.2`. Python3 is installed on the computer but the command starts with `python` rather than `python3`.  

It is pretty ridiculous that when I saw the exception, I checked the `plugins/` directory and didn't see the `worddumb-libs` directory. However, when I executed your command without doing anything else, it worked. 🤨
